### PR TITLE
Trinary Perfection clothing

### PR DESCRIPTION
**Pull request by TheGoret**
On the occasion of The Trinary Perfection rework, a religion tab has been added to the loadout. All clothes related to this religion are available in this new tab.
In the future, it can also be used for other species religion.

Lore Request. Thanks to Scrambles for the sprites and Abo for the help he gave me.

### CODE REVIEW

**Comment by TheGoret** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm [hunk 1]
```diff
@@ -76,6 +76,7 @@
 	antenna["antenna, double"] = /obj/item/clothing/head/antenna/double
 	antenna["antenna, left"] = /obj/item/clothing/head/antenna/double/left
 	antenna["antenna, right"] = /obj/item/clothing/head/antenna/double/right
+	antenna["antenna, triniary perfection"] = /obj/item/clothing/head/antenna/trinary_halo
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm]:
```suggestion
	antenna["antenna, trinary perfection"] = /obj/item/clothing/head/antenna/trinary_halo
```

File: code/modules/client/preference_setup/loadout/loadout_religion.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/datum/gear/religion
+	display_name = "triniary perfection robes"
```
Comment (ParadoxSpace) [inline code/modules/client/preference_setup/loadout/loadout_religion.dm]: triniary

File: code/modules/client/preference_setup/loadout/loadout_religion.dm [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	path = /obj/item/clothing/suit/trinary_robes
+	sort_category = "Religion"
+
+/datum/gear/religion/triniary/mask
```
Comment (ParadoxSpace) [inline code/modules/client/preference_setup/loadout/loadout_religion.dm]: also here

File: code/modules/client/preference_setup/loadout/loadout_religion.dm [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+	display_name = "triniary perfection mask"
+	path = /obj/item/clothing/mask/trinary_mask
+
+/datum/gear/religion/triniary/cape
+	display_name = "triniary perfection cape"
```
Comment (ParadoxSpace) [inline code/modules/client/preference_setup/loadout/loadout_religion.dm]: also here

File: code/modules/client/preference_setup/loadout/loadout_religion.dm [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	path = /obj/item/clothing/accessory/poncho/trinary
+
+/datum/gear/religion/triniary/badge
```
Comment (ParadoxSpace) [inline code/modules/client/preference_setup/loadout/loadout_religion.dm]: also here

File: code/modules/clothing/masks/miscellaneous.dm [hunk 6]
```diff
@@ -92,6 +92,17 @@
 	flags_inv = HIDEFACE
 	body_parts_covered = 0
 
+/obj/item/clothing/mask/trinary_mask
+	name = "trinary perfection mask"
+	desc = "A simple lace mask worn by IPCs and organics alike while within the churches of the Trinary Perfection."
+	icon_state = "trinary_mask"
+	item_state = "trinary_mask"
+	w_class = 2
```
Comment (mikomyazaki) [inline code/modules/clothing/masks/miscellaneous.dm]:
```suggestion
	w_class = ITEMSIZE_SMALL
```

File: html/changelogs/goret-robotman-religion.yml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "A religion tab in the loadout menu has been added. It could be used in the future for religious stuff."
```
Comment (mikomyazaki) [inline html/changelogs/goret-robotman-religion.yml]: It is being used immediately for religious stuff, by you. This line makes it sound like it is unused.

File: code/modules/client/preference_setup/loadout/loadout_religion.dm [hunk 8]
```diff
@@ -0,0 +1,20 @@
+/datum/gear/religion
+	display_name = "trinary perfection robes"
+	path = /obj/item/clothing/suit/trinary_robes
+	sort_category = "Religion"
+
+/datum/gear/religion/triniary/mask
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_religion.dm]:
```suggestion
/datum/gear/religion/trinary/mask
```

File: code/modules/client/preference_setup/loadout/loadout_religion.dm [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+	display_name = "trinary perfection mask"
+	path = /obj/item/clothing/mask/trinary_mask
+
+/datum/gear/religion/triniary/cape
+	display_name = "trinary perfection cape"
+	path = /obj/item/clothing/accessory/poncho/trinary
+
+/datum/gear/religion/triniary/badge
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_religion.dm]:
```suggestion
/datum/gear/religion/trinary/badge
```

File: code/modules/client/preference_setup/loadout/loadout_religion.dm [hunk 10]
```diff
@@ -0,0 +1,20 @@
+/datum/gear/religion
+	display_name = "trinary perfection robes"
+	path = /obj/item/clothing/suit/trinary_robes
+	sort_category = "Religion"
+
+/datum/gear/religion/triniary/mask
+	display_name = "trinary perfection mask"
+	path = /obj/item/clothing/mask/trinary_mask
+
+/datum/gear/religion/triniary/cape
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_religion.dm]:
```suggestion
/datum/gear/religion/trinary/cape
```

File: code/modules/clothing/masks/miscellaneous.dm (hunk 6 above)
Comment (mikomyazaki) [inline code/modules/clothing/masks/miscellaneous.dm]:
```suggestion
	w_class = ITEMSIZE_SMALL
```

You reverted this change for some raisin?

Comment (TheGoret) [reply code/modules/clothing/masks/miscellaneous.dm]: Your thing is outdated sir!

Comment (mikomyazaki) [reply code/modules/clothing/masks/miscellaneous.dm]: I blame github's website... somehow. :+1:

**Pull request merged**